source/codegen/templates/service.cpp.mako

```diff
@@ -45,6 +45,17 @@ namespace ${config["namespace_component"]}_grpc {
     }
   }
 % endif
+% if common_helpers.has_enum_array_string_out_param(functions):
+  template <typename T1, typename T2>
```

Comment:
Since we are writing this function specifically for string params, can we make first parameter as `std::string` instead of generic T1?

Reply:
I made the changes as we discussed yesterday, so using the T1 * as first parameter.